app/controllers/submission/workflow/files.js

```diff
@@ -0,0 +1,23 @@
+import Controller from '@ember/controller';
+import { computed } from '@ember/object';
+import { alias } from '@ember/object/computed';
+
+export default Controller.extend({
+  submission: alias('model.submission'),
+
+  files: computed('submission', function () {
+    return this.get('submission.files');
```

Comment:
Generally I think using imported version of `get(this, ...` is preferred rather than `this.get ...`. There's a codmod for it, but the basic idea is you can use `this.get` in most places, but if there's a promise proxy use the imported `get`.

https://blog.emberjs.com/2018/04/13/ember-3-1-released.html#toc_es5-getters-for-computed-properties-2-of-4